debezium-core/src/main/java/io/debezium/transforms/debeziumtimestampcoverter/TimestampTranslator.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.transforms.debeziumtimestampcoverter;
+
+import org.apache.kafka.connect.data.Schema;
+
+import java.util.Date;
+
+/**
+ * DebeziumTimestampConverter uses this interface to define custom ways of converting different versions/types of timestamp/date formats.
+ * DebeziumTimestampConverter has a static block that collects all the custom conversion objects into TRANSLATORS map,
+ * and are called based the timestamp type conversion.
+ */
+
+public interface TimestampTranslator {
```

Comment:
Perhaps this should be defined as a static interface within `DebeziumTimestampConverter`, and could then be called `Translator` as it is then scoped within `DebeziumTimestampConverter`.